components/cmp/services/test/getConsentStatusSpec.js

```diff
@@ -13,12 +14,16 @@ describe('getConsentStatus', () => {
     window.__cmp.restore()
   })
 
-  it('Should return you if the user has accepted the cookies', () => {
-    getConsentStatus.execute().then(response => {
-      expect(window.__cmp.calledOnce).to.equal(true)
-      const [command] = window.__cmp.getCall(1).args
-      expect(command).to.equal('getConsentStatus')
-      expect(response.userAcceptedCookies).to.be(false)
-    })
+  it('Should return you if the user has accepted the cookies', done => {
+    getConsentStatus
+      .execute()
+      .then(response => {
+        expect(window.__cmp.calledOnce).to.be.true
+        const command = window.__cmp.getCall(0).args[0]
+        expect(command).to.equal('getConsentStatus')
+        expect(response).to.equals(CONSENT_STATUS_NOT_ACCEPTED)
+        done()
+      })
+      .catch(error => done(new Error(error)))
```

Comment:
Wouldn't it do the same if you just do `.catch(done)`?. I'm not sure, but I thought that if you call done with an error (which will happen when ending up in the catch) the test will automatically fail

Reply:
Nop, the last time that I checked this, if value from catch is false for example, calling function "done(false)" will do the test green.
Seems that done function must get an Error object to fail.